crates/node/src/evm.rs

```diff
@@ -28,7 +28,7 @@ use reth_revm::{
 use std::sync::Arc;
 
 /// Custom EVM configuration
-#[derive(Debug, Clone, Default)]
```

Comment:
removed default so we don't accidentally use it and end up with a bogus chainspec